components/02-molecules/pull-quote/pull-quote.twig

```diff
@@ -0,0 +1,34 @@
+{#
+ # Available Props:
+ # - pull_quote__style: bar-left, bar-right, or quote-left
+ #
+ # Available Variables:
+ # - pull_quote__quote (required)
+ # - pull_quote__attribution
+ #}
+
+{% set pull_quote__base_class = 'pull-quote' %}
+{% set pull_quote__style = pull_quote__style|default('bar-left') %}
+
+{# If pull_quote__attributes is not defined, set it to an empty object by default #}
+{% set pull_quote__attributes = pull_quote__attributes|default({}) %}
+
+{% set pull_quote__attributes = pull_quote__attributes|merge({
+  'data-component-width': 'content',
+  'data-pull-quote-style': pull_quote__style,
+  class: bem(pull_quote__base_class),
+}) %}
+
+<div {{ add_attributes(pull_quote__attributes) }}>
```

Comment:
I've read that a pull quote is a good candidate for the `<aside>` tag -- to mark it as being supplemental to the content and not interrupt the flow. https://htmldog.com/techniques/pullquotes/